### DISCUSSION TITLE
(Better) explanation for using Nerdfonts on Websites.

### DISCUSSION THREAD

**Discussion by paulfrische** [ryanoasis/nerd-fonts#866] [Ideas]
Hello there, I'd really like to use Nerdfonts on my website but because of the lack of explanations how  to use them as such I am not able to do so. I think a (better) explanation would help many people including me.
Have a great day!

**Comment by Finii** [comment]
I believe there is nothing here, technically, that allows it (i.e. is impossible).

But my knowledge of 'fetching remote fonts on a web-page' is rather slim. Any suggestions welcome.

**Comment by paulfrische** [comment]
but in the cheat sheet there is the following example usage for css/html:
```html
I really <i class="nf nf-fa-heart"></i> <i class="nf nf-custom-vim"></i>
```

**Reply by Finii** [reply]
Ah, never noticed that ...

![Screenshot from 2022-07-16 13-19-45](https://user-images.githubusercontent.com/16012374/179352774-ba5225c8-d951-4ff0-85fc-543c470a78b8.png)

And if one allows the page to do scripting it even loads the font from somewhere...

![image](https://user-images.githubusercontent.com/16012374/179352940-5b8b515b-9eed-4091-985c-65a8fc0875a0.png)

Ok, thats a starting point...

**Reply by Finii** [reply]
I guess you need to
* take this css: https://github.com/ryanoasis/nerd-fonts/blob/gh-pages/_includes/css/nerd-fonts.css
* put the font you want to use somewhere in the web tree
* correct the path to the font (and the font-name, maybe) in the `nerd-fonts.css`

Report back if that works ;-)
